### DISCUSSION TITLE
Error: Invalid credentials. Invalid username or password.

### DISCUSSION THREAD

**Discussion by hahahabirdx** [PrismarineJS/mineflayer#2433] [Q&A]
- [x] The [FAQ](https://github.com/PrismarineJS/mineflayer/blob/master/docs/FAQ.md) doesn't contain a resolution to my issue 

Was testing my bot and everything was fine. Bot was working. When I restarted it came up with **_"Error: Invalid credentials. Invalid username or password."_** My info is right and everything and I have a mojang account. Was my account blacklisted by Yggdrasil for restarting the bot too many times or something? Anyways thanks!

info below
![image](https://user-images.githubusercontent.com/78945353/151467931-07d851dd-53a7-419a-bcc8-7126b2fe24d9.png)

**Comment by IceTank** [comment]
If you are 100% sure your login information is right then yes you may have been blacklisted from mojang login. You can try:
1. Logging into minecraft.net with that account and then try again
2. Using a different ip
3. Just waiting a few hours
4. Email support to unblock your account

**Reply by rom1504** [reply]
@IceTank do you feel like adding that information somewhere in node-minecraft-protocol doc, and then adding a link to that doc in the error thrown by minecraft-protocol?
I think it would help people
